getting-started-guides/csp/gcp/gcp.md

```diff
@@ -88,9 +91,9 @@ gcloud beta dataproc clusters create $CLUSTER_NAME  \
     --num-worker-local-ssds 1 \
     --num-workers $NUM_WORKERS \
     --image-version 1.4-ubuntu18 \
-    --bucket $STORAGE_BUCKET \
-    --metadata JUPYTER_PORT=80,INIT_ACTIONS_REPO="gs://$STORAGE_BUCKET",linux-dist="ubuntu" \
-    --initialization-actions gs://$STORAGE_BUCKET/spark-gpu/rapids.sh \
+    --bucket $GCS_BUCKET \
+    --metadata JUPYTER_PORT=8123,INIT_ACTIONS_REPO="gs://$INIT_ACTIONS_BUCKET",linux-dist="ubuntu",GCS_BUCKET="gs://$GCS_BUCKET" \
+    --initialization-actions gs://$INIT_ACTIONS_BUCKET/spark-gpu/rapids.sh \
     --optional-components=ANACONDA,JUPYTER \
     --subnet=default \
     --properties '^#^spark:spark.dynamicAllocation.enabled=false#spark:spark.shuffle.service.enabled=false#spark:spark.submit.pyFiles=/usr/lib/spark/python/lib/xgboost4j-spark_2.11-1.0.0-Beta2.jar#spark:spark.jars=/usr/lib/spark/jars/xgboost4j-spark_2.11-1.0.0-Beta2.jar,/usr/lib/spark/jars/xgboost4j_2.11-1.0.0-Beta2.jar,/usr/lib/spark/jars/cudf-0.9.1-cuda10.jar' \
```

Comment:
Beta3 updates

Reply:
cudf-0.9.1-cuda10.jar should be cudf-0.9.2-cuda10.jar

Reply:
done, also need to change maven url and from `xgboost4j_2.11` to `xgboost4j_2.x`